tests/BlazingSQLTest/Runner/config.yaml

```diff
@@ -0,0 +1,40 @@
+TEST_SETTINGS:
+#  DATA_DIRECTORY: $CONDA_PREFIX/blazingsql-testing-files/data/ - Not yet supported in yaml.
+#  FILE_RESULT_DIRECTORY: $CONDA_PREFIX/blazingsql-testing-files/results/ - Not yet supported in yaml.
+  LOG_DIRECTORY: $CONDA_PREFIX/
+  DATA_SIZE: "100MB2Part"
+  EXECUTION_ENV: "local"
+  DASK_CONNECTION: "local" # values: "dask-scheduler-ip:port", "local"
+
+RUN_SETTINGS:
+#  SAVE_LOG: "false" - Not yet supported in yaml.
+  EXEC_MODE: gpuci #gpuci, full, generator
+  NGPUS: 1
+  NETWORK_INTERFACE: lo
+  WORKSHEET: BSQL Log Results # or "BSQL Performance Results"
+  LOG_INFO:
+  CONCURRENT: False
+#  CONCURRENT:  - Not yet supported in yaml.
```

Comment:
not supported? but the line above looks like it is?

Reply:
The plan is to support multiple configurations at the same time. At the moment only one setting works for that parameter. For example, for the following configuration:
``` yaml
CONCURRENT:
   - False
   - True 
```
Then you will run two sets of tests, one for `CONCURRENT = True` and one for `CONCURRENT = False`, and thus simplify `test.sh` at this point. This part not yet support.